create_diagram/create_diagram.py

```diff
@@ -110,13 +103,28 @@ def create_diagram(table_name, dataframe, primary_keys = ["WorkerId", "round_num
         diagram_creator += f"  PRIMARY KEY ({pk_string})\n"
 
     diagram_creator += ");"
-    
+
     # Print the variable names that were reserved keywords
-    changed_varnames_str = ",\n ".join(changed_varnames)    
-    print(f"Variable names that were found to be keywords and subsequently modified by adding _var: {changed_varnames_str}\n\n")
-    
-    
+    changed_varnames_str = ",\n ".join(changed_varnames)
+    print(
+        f"Variable names that were found to be keywords and subsequently modified by adding _var: {changed_varnames_str}\n\n")
+
     return diagram_creator
-    
-    
-    
\ No newline at end of file
+
+
+if __name__ == '__main__':
+
+    table_name = input("Input your table name: ")
+
+    dataframe_path = input("Input your dataframe file path: ")
+
+    if dataframe_path.endswith(".csv"):
+        dataframe = pd.read_csv(dataframe_path)
+    elif dataframe_path.endswith((".xlsx", ".xls")):
+        dataframe = pd.read_excel(dataframe_path)
+
+    pre_primary_keys = input("Enter primary keys (if multiple, separate them by a space): ")
+    primary_keys = pre_primary_keys.split()
+
+    sql_commands = create_diagram(table_name, dataframe, primary_keys)
+    print(sql_commands)
```

Comment:
@wagnerlmichael this is just printing the commands, not storing them on a txt file. We need to store the output on a txt. It could have the same name as the table

Reply:
Got it, I'll fix for tomorrow

Reply:
@mgutierrezc Added txt

Reply:
great! could you erase the print statement of SQL commands?

Reply:
done!